src/api/plugin-api.ts

```diff
@@ -608,6 +608,8 @@ export type QueryApiSettings = {
 
 /** Determines if source-path has a `?no-dataview` annotation that disables dataview. */
 export function isDataviewDisabled(sourcePath: string): boolean {
+    if (sourcePath == null) return false;
```

Comment:
Consider checking for falsy value instead.

```js
if (!sourcePath) return false;
```

Reply:
Done